### PR TITLE
fs/smartfs : Add erase info of journal block

### PR DESCRIPTION
**Pull request by Taejun-Kwon**
If CONFIG_MTD_SMART_SECTOR_ERASE_DEBUG set, it will show erase count of journal block

### CODE REVIEW

**Review by jsdosa — APPROVED** [review]
looks nice to me.

**Review by sunghan-chang — APPROVED** [review]

File: os/fs/driver/mtd/smart.c [hunk 1]
```diff
@@ -4894,11 +4905,23 @@ static int smart_journal_move_to_next(FAR struct smart_struct_s *dev)
 	/* Check we have to move to next block, delete current block if it is full */
 	if (((old_seq + 1) % dev->njournalPerBlk) == 0) {
 		old_block = old_seq / dev->njournalPerBlk;
-		ret = smart_journal_erase(dev, old_block + dev->neraseblocks);
+		old_block += dev->neraseblocks;
```
Comment (amogh-samsung) [inline os/fs/driver/mtd/smart.c:4908]: Could you clarify why this is added?

**Review by amogh-samsung — COMMENTED** [review]
Rest all looks good to me!

File: os/fs/driver/mtd/smart.c (hunk 1 above)
Comment (Taejun-Kwon) [reply os/fs/driver/mtd/smart.c:4908]: If you meaning dev->neraseblocks, then dev->neraseblocks is MTD block driver's area.
From dev->neraseblocks, journal area started.
To get entire of eraseblocks that is allocated to smartfs, you can use dev->geo.eraseblocks.

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Pull request merged**